pxr/usdImaging/usdImaging/delegate.h

```diff
@@ -659,7 +659,13 @@ class UsdImagingDelegate : public HdSceneDelegate, public TfWeakBase {
     _DependencyMap _dependencyInfo;
 
     void _GatherDependencies(SdfPath const& subtree,
-                             SdfPathVector *affectedCachePaths);
```

Comment:
As with _GatherDependenciesCache, our code convention is to not have data written out to reference parameters.